pkg/volume/empty_dir/empty_dir.go

```diff
@@ -169,14 +169,14 @@ func (ed *emptyDir) SetUpAt(dir string) error {
 }
 
 func (ed *emptyDir) setupDefault(dir string) error {
-	return os.MkdirAll(dir, 0750)
+	return os.MkdirAll(dir, 0777)
```

Comment:
And tomorrows 1666 will break every programming language ever that treats a leading 0 as octal.

----- Original Message -----
> > @@ -169,14 +169,14 @@ func (ed *emptyDir) SetUpAt(dir string) error {
> >  }
> >  
> >  func (ed *emptyDir) setupDefault(dir string) error {
> > -	return os.MkdirAll(dir, 0750)
> > +	return os.MkdirAll(dir, 0777)
> 
> are there any real constants in this world?  today's 0777 might be tomorrow's
> 1666.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/9384/files#r31918052